test/attempts.test.js

```diff
@@ -23,4 +24,19 @@ describe('Request attempts', function () {
       done();
     });
   });
+
+  it('should call delay strategy 2 times after some retries', function (done) {
+    var mockDelayStrategy = sinon.stub().returns(100);
+
+    request({
+      url: 'http://www.filltext.com/?rows=1&err=500', // return a 500 status
+      maxAttempts: 3,
+      retryDelay: 100,
```

Comment:
`retryDelay` should be something like `100000` (so it will make the test fail)
and then we would have to check that the delay between "before the request" and "the callback was called" is at least 300ms, so we will be sure the delayStrategy was used. What do you think @oztheozoz ?